geoips/schema/bases/valid_identifier.yaml

```diff
@@ -1,4 +1,4 @@
 $id: bases.valid_identifier
 type: string
 description: "A valid Python identifier"
-pattern: "^(?!.*(\\b|_)(?:class|def|if|else|elif|while|for|try|except|finally|with|import|from|as|global|nonlocal|lambda|and|or|not|in|is|assert|True|False|None)\\b)[a-zA-Z_][a-zA-Z0-9_]*$"
+# pattern: "^(?!.*(\\b|_)(?:class|def|if|else|elif|while|for|try|except|finally|with|import|from|as|global|nonlocal|lambda|and|or|not|in|is|assert|True|False|None)\\b)[a-zA-Z_][a-zA-Z0-9_]*$"
```

Comment:
This is commented out since some products start with a number ?

Reply:
Yes. For the time being I've decided to just quit checking this in general. I would like to get back to checking this, though, and would like to enforce this for all names. It will make some things easier in the future.

Do you think it would be unreasonable to require this for product names?